Re-add .NET Standard 2.0 to framework targets…

papeh opened the pull request:
…for the projects used by LFMerge.

Also reference Win32.Registry in LibTriboroughBridge-ChorusPlugin to fix a pretty ugly build warning about version conflicts.

This reverts commit d00ed11f584e81ebf8f2c2fe7670bd7f83114180.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/sillsdev/flexbridge/372)
<!-- Reviewable:end -->

papeh committed:
Re-add .NET Standard 2.0 to framework targets for the projects used by LFMerge.

Also reference Win32.Registry in LibTriboroughBridge-ChorusPlugin to fix a pretty ugly build warning about version conflicts.

This reverts commit d00ed11f584e81ebf8f2c2fe7670bd7f83114180.

josephmyers commented:
Without having looked at this a ton, my understanding is this is blocked until we get [the Chorus PR](https://github.com/sillsdev/chorus/pull/279) in.

josephmyers commented:
This is unblocked now

ermshiperete reviewed (changes requested):
This should have an entry in the Added section of `CHANGELOG.md` and a `+semver:minor` in the commit message.

josephmyers committed:
+semver:minor

Updated changelog

josephmyers committed:
Merge branch 'develop' into feature/net-standard

ermshiperete reviewed (approved):
Are you planning on creating a 3.6.4 release today? Otherwise don't create the 3.6.4 section head but leave it under Unreleased.

josephmyers commented:
Oh got it

josephmyers committed:
Removed accidental release tag in changelog

papeh reviewed (commented):
For some reason, TeamCity jobs ([Windows](https://build.palaso.org/buildConfiguration/FLExBridge_FLExBridgeDevelopWin32Continuous) and [Linux](https://build.palaso.org/buildConfiguration/FLExBridgeDevelopLinux64Continuous)) are unable to see this PR. Strange that problem seems to be consistent for and peculiar to .NET Standard PR's. Since we still use TeamCity for Windows release builds, I hope we can get this straightened out.

> *[Reviewable](https://reviewable.io/reviews/sillsdev/flexbridge/372#-:-NAf_Rsh9rf3EeCwmkrH:b-7ibe1x)* status: 0 of 4 files reviewed, all discussions resolved (waiting on @josephmyers)



<!-- Sent from Reviewable.io -->

josephmyers committed:
Using Chorus 5.0.1-beta* instead of 5.0.0-beta*

This is required now that Chorus released its 5.0.0 version. 5.0.1 is required to reference the .NET Standard versions.

josephmyers commented:
I see that the builds still run on develop. I can test that the bug is present by opening a new branch and PR, but I don't have the ability to do much on the TC configuration side.

Edit: Yep, [branch off of develop](https://github.com/sillsdev/flexbridge/pull/374) does not trigger the TC builds. I will say that the libpalaso builds are using .NET Standard with TeamCity successfully. Maybe someone can compare the configs?
